expo/gbe/migrations/0008_auto__add_usermessage__add_unique_usermessage_view_code.py

```diff
@@ -0,0 +1,387 @@
+# -*- coding: utf-8 -*-
```

Comment:
Okay, I'm going to call this a "whatever". Either we're going to get up to supported django and delete all of these migrations, or we're not and it won't matter. Not a blocker.